catchpoint-mixin/targets.libsonnet

```diff
@@ -9,4 +9,111 @@ local utils = commonlib.utils {
   new(this): {
     local vars = this.grafana.variables,
     local config = this.config,
+    local testNameLabel = config.instanceLabels + config.testNameLabel,
+
+    topAvgLoadTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by (test_name) (2, avg by (instance, test_name) (catchpoint_load_time{%(testNameSelector)s}))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topMaxTotalLoadTime:
```

Comment:
This query should not be grouped by test name here. It is looking for the slowest total load time, and should have a test name and node name in the legend